### ISSUE TITLE
Fokus på link, navlink og tertiærknapp

### ISSUE THREAD

**Issue by Murstam** [fremtind/jokul#1457]
Egen focus state for å synliggjøre hvor man er ved bruk av tabbing.
https://www.figma.com/file/TkbB9ANfejDSjB2u4u1OEuqM/J%C3%B8kul-komponenter-byggesone?node-id=17521%3A50308

**Comment by Steinop** [comment]
På en måte liker jeg 2px avrundingen fordi den myker opp uttrykket litt og gir en liten assosiasjon til pilleformen til knappene og viderefører avrundingen på inputfeltene. De innbyr til å trykkes på. Samtidig lurer jeg på om vi bør være tøffere og kjøre på med helt skarpe kanter men det er et valg vi må ta. Kanskje noe vi kan diskutere i VD-forum eller legge ut på Teams for flere synspunkter/forankring?

Ellers et par ting som bør vurderes:
1. Fokusmarkering på tertiærknapp bør følge samme visuelle språk som de to andre slik at den ikke ser ut som en sekundærknapp.
2. Fokusmarkeringen på lenke bør holdes innenfor linjehøyden slik at rammen ikke krasjer med tekst på linjen under i løpende tekst.

Har laget et par raske utkast (som innspill) rett nedenfor dine skisser @Murstam

**Comment by Murstam** [comment]
Ja, det var tanken med 2px avrunding. Jeg er egentlig enig, la oss gå for skarpe kanter. Har lagt stroke på innsiden nå og strammet opp tertiærknappen. Ligger til høyre for forrige utkast. @Steinop

**Comment by Steinop** [comment]
Topp! Bare én liten pirkedetalj: 
Tertiærknappen vokser 5% (x1,05) på focus slik den gjør på hover. Enten bør rammen følge forstørrelsen evt bør vi fjerne forstørrelsen på focus på primær, sekundær og tertiær.

**Comment by Murstam** [comment]
Nå har jeg ganget alle verdier med 1,05, men det stemmer ikke helt overens med hoverstørrelse?

**Comment by Steinop** [comment]
Interessant. @Mikkelinski lagde disse basert på versjonen i kode. Vi må uansett gå gjennom disse komponentene på nytt når Figma kommer med sin etterlengtede oppdatering med komponentstates. Høyde 50.4 px for knapper som er 48 px stemmer med min kalkulator 😁

**Comment by Murstam** [comment]
50,4 er riktig, men den er 50,84px i Figma. 

Men da er hvertfall ny fokus klar og da er det bare å få det inn i Figma og Jøkul. Hvem tar det?

**Comment by Steinop** [comment]
Jeg kan bidra med oppdatering av komponenten i Figmabiblioteket. Har dere utviklere som kan bidra på kodesiden?

**Comment by Steinop** [comment]
Komponentene er opptatert i Figmabiblioteket. 

Lenker og eksterne lenker er bygget opp fra bunn i Figma og forenklet strukturmessig. Tidligere var komponenten 1px for høy pga en bug i Autolayout. Nå matcher normalversjonen en linjehøyde på 32 px mens kompakt er 28 px høy. Redesignet ikon for ekstern lenke er også lagt til i komponenten.

Normalt sett vil vi kun bruke understrek i løpende tekst som markering for lenker når vi designer fordi det blir tungvint å bryte opp tekst for å legge inn denne. Likevel er den tilgjengelig i biblioteket til bruk på frittstående lenker og som referanse for utvikler.

**Comment by Murstam** [comment]
Supert. Lite kapasitet på utviklerne i BM-teamet, så blir nok ikke tatt tak i av dem med det første.

**Comment by Steinop** [comment]
OK. Da ligger den uansett klar til utvikling når noen føler seg kallet 😁
